utils/sortJson.js

```diff
@@ -0,0 +1,19 @@
+const json = require("../data/gist.json");
```

Comment:
What do you think to have the relative path as an argument with progress.argv?
like:
```sh
node ./sort.js gist.json
```

Reply:
I just thought this could be a naming convention for now.